.github/workflows/conda_cron.yaml

```diff
@@ -0,0 +1,54 @@
+name: "conda_cron"
+on:
+  schedule:
+    # At 07:00 UTC every day
+    - cron: "0 7 * * *"
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  condacheck:
```

Comment:
This should hopefully meet everything on your wishlist @richardjgowers - including a rough "raise an issue if it fails" (which we can iterate though / refine at a later point if folks want dashboards, etc...).